lib/index.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
     'unused-module': require('./rules/unused-module')
   },
   configs: {
+    app: require('./configs/app'),
     browser: require('./configs/browser'),
```

Comment:
Since we use the `browser` rule set as the "app" today, my hunch is that we want to add the `require-await` rule to `browser` and introduce a new `lib` or `library` rule set that disables that rule.